recipes/dask-mpi/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "dask-mpi" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 77afcf670017c2ef2139732091b743de96ff4a122fe7156275c84b4df745389c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
Would add the following...

```yaml
  entry_points:
    - dask-mpi = dask_mpi.cli:go
```

Reply:
Would add this change should allow `noarch: python` to be added back.

Reply:
Thanks, done!